# FaceBoundingBoxArray.msg
# 多个人脸的 bounding box 列表

FaceBoundingBox[] faces
